spinalcordtoolbox/scripts/sct_analyze_lesion.py

```diff
@@ -333,6 +335,32 @@ def _measure_axial_damage_ratio(self, im_data, p_lst, idx):
         printv('  Maximum axial damage ratio: ' + str(np.round(maximum_axial_damage_ratio, 2)),
                self.verbose, type='info')
 
+    def _compute_issue_bridge_ratio(self, idx):
```

Comment:
nitpick, but: typo!

```suggestion
    def _compute_tissue_bridge_ratio(self, idx):
```

(will have to fix the caller site as well)